libs/components/forms/src/lib/modules/radio/radio-group.component.spec.ts

```diff
@@ -170,23 +178,29 @@ describe('Radio group component (reactive)', function () {
   }));
 
   it('should not show a required state when not required', fakeAsync(() => {
+    componentInstance.labelText = 'Test';
```

Comment:
with this added here, are we testing that this functionality still works when consumers are not opted in to easy mode (ie with label text set)?

Reply:
"Hard mode" for this component would require the consumer to pass in the `label` or `legend` element directly - at which point the setting of the asterisk would be on them and not us. We don't have a separate label component for this component like we did for some of the others.